.releaserc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "tagFormat": "${version}",
+  "branches": [
+    "main",
+    "shahroz/ci-cd-deployment"
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    ["@semantic-release/exec", {
+      "prepareCmd": "bundle exec fastlane android deploy_app version:${nextRelease.version}"
```

Comment:
should the iOS app also be deployed here? Instead of only the Android app?

Reply:
Good catch !! Thank you.